spec.md

```diff
@@ -8,24 +8,24 @@ interpreted as described in [RFC 2119][].
 
 ## Overview
 
-This specification extends, expands and replaces [PSR-2][], the coding style guide and
+This specification is a live document that extends, expands and replaces [PSR-12][], the extended coding style guide and
```

Comment:
In my opinion this document is not a living document. The latest commit is not the source of truth the CC approved tag is.

Reply:
Makes sense. Will fix that.